src/components/molecules/InternalChevronAccordion/InternalChevronAccordion.stories.tsx

```diff
@@ -4,16 +4,16 @@ import styled from "styled-components";
 
 import { OakHandDrawnHR } from "../OakHandDrawnHR";
 
-import { InternalShadowAccordion } from "./InternalShadowAccordion";
+import { InternalChevronAccordion } from "./InternalChevronAccordion";
```

Comment:
Seeing as there is no shadow this name makes more sense. I do wonder whether the 3 layers are necessary. Perhaps this could have been wrapped up with the InternalAccordian ?